profiling/src/profiling/mod.rs

```diff
@@ -1000,6 +1000,24 @@ impl Profiler {
 
     fn message_labels() -> Vec<Label> {
         let gpc = unsafe { datadog_php_profiling_get_profiling_context };
+
+        let mut labels = vec![
+            Label {
+                key: "thread id",
+                value: LabelValue::Num(unsafe { libc::pthread_self() as i64 }, "id".into()),
+            },
+            Label {
+                key: "thread name",
+                value: LabelValue::Str(
+                    thread::current()
+                        .name()
+                        .unwrap_or("Main")
+                        .to_string()
+                        .into(),
```

Comment:
I'm pretty sure PHP doesn't name its threads on any usable level in web SAPIs. In ZTS, this means we're going to get like 4+ threads called "Main".  Maybe we should call this "Thread {tid}"?

Second, I don't know if Rust's thread names are what we want. I know that Rust conveys thread names to the OS, but I don't think the reverse happens. If someone actually names a thread with `pthread_setname_np`, will Rust pick it up? We should check.

Reply:
It doesn't.